README.md

```diff
@@ -26,8 +26,9 @@ NativeJDB uses the [JDWP protocol](https://docs.oracle.com/en/java/javase/11/doc
     - Clear Breakpoints (Delete/Disable)
     - Step Over / Step Into / Step Return
     - Stack Frames information in IDE debugger pane
-    - Variables (Local + Static) values (Work-in-progress)
-    - View of assembly code within a stack frame (work in progress)
+    - Primitive variables (Local + Static) values (Work-in-progress)
```

Comment:
Can you remove the `(Work-in-progress)` for Primitive variables since we finished that?